docs/workflows/best-practices-organization.md

```diff
@@ -8,12 +8,13 @@ The purpose of this document is to provide concrete and actionable guidelines on
 
 ## Three stakeholder groups in a hybrid model
 
-Typically, nobody in an organization is opposed to building quality software. Rather, investing in software quality often loses out to other topics, typically adding features and fixing bugs. The shift that needs to take place is that organizations allow for a fair competition to take place between feature work and technical improvements. 
+Typically, nobody in an organization is opposed to building quality software. Rather, investing in software quality often loses out to other topics, typically adding features and fixing bugs. The shift that needs to take place is that organization arrive at a way of working that allows for a fair competition to take place between feature work and technical improvements. 
 
 Having consulted with many organizations, we have identified three stakeholder groups that need to be involved to make this shift and achieve high quality software, structurally. These are:
 - Teams
 - Engineering leads
-- Management
+- The User Organization
```

Comment:
Given the focus on building software, we now (appear to) leave out operational roles, like service manager, sysadmin, ops engineer. I don't think that should be yet another group, but since perceived quality of software also depends on how it's actually run, non-SIG readers might expect us to say something about it. Perhaps we should mention this explicitly as out of scope for this model?

Reply:
I consider the teams to be devops teams, so they would include all roles needed to provide the service. In practice, separate ops organizations exist. If you'd try to include them in the picture, it becomes clear why people have been moving to devops: if separate ops teams exist, they would either be 'stuck' with whatever the dev team provides, or they would have to go through a product owner to get things on the backlog or else open a 'backchannel' to the team to do so. None of this is great. But I can indeed mention this.

Reply:
Made a mention of this.